spec/spec.html

```diff
@@ -1,25 +1,30 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <title>Mocha</title>
-    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="../node_modules/mocha/mocha.css" />
-  </head>
-  <body>
-    <div id="mocha"></div>
-    <script src="../node_modules/mocha/mocha.js"></script>
-    <script src="../node_modules/chai/chai.js"></script>
-    <script src="../node_modules/knockout/build/output/knockout-latest.js"></script>
-    <!-- Application code goes here -->
-    <script src="../lib/calculator.js"></script>
 
-    <script>mocha.setup('bdd');</script>
+<head>
+  <title>Mocha</title>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="../node_modules/mocha/mocha.css" />
+</head>
 
-    <!-- Test scripts below -->
-    <script src="./calculator.spec.js"></script>
+<body>
+  <div id="mocha"></div>
+  <script src="../node_modules/mocha/mocha.js"></script>
+  <script src="../node_modules/requirejs/require.js"></script>
+  <script>mocha.setup('bdd');</script>
 
-    <!-- mocha.run() should be last -->
-    <script>mocha.run();</script>
-  </body>
-</html>
+  <script>
+    require.config({
+      paths: {
+        'chai': '../node_modules/chai/chai',
+        'knockout': '../node_modules/knockout/build/output/knockout-latest'
+      }
+    });
+    require(['./calculator.spec.js'], function() {
```

Comment:
Here we tell RequireJS to load our spec files (an array of strings, but in this case we only have 1 file). Once the files have loaded, the callback will be called (remember all the work on callbacks?). Inside that callback we call `mocha.run()`.